src/navigation/drive.py

```diff
@@ -21,17 +24,17 @@ def get_drive_command(target_pos : np.ndarray, rover_pos : np.ndarray, rover_dir
 
     if target_dist < completion_tolerance:
        return Twist(), True
-    else:
-        cmd_vel = Twist()
-        if alignment > DRIVE_FWD_THRESH:
-            # We are pretty aligned so we can drive straight
-            error = target_dist
-            cmd_vel.linear.x = np.clip(error, 0.0, 1.0)
-        # Determine the sign of our effort by seeing if we are to the left or to the right of the target
-        # This is done by dotting rover_dir and target_dir rotated 90 degrees ccw
-        perp_alignment = rover_dir[0] * -target_dir[1] + rover_dir[1] * target_dir[0]
-        sign = -np.sign(perp_alignment)
-        # 1 is target alignment
-        error = 1.0 - alignment
-        cmd_vel.angular.z = np.clip(error * 100.0 * sign, -1.0, 1.0)
-        return cmd_vel, False
\ No newline at end of file
+    
+    cmd_vel = Twist()
+    if alignment > turn_in_place_tolerance_rad:
```

Comment:
It's not rad - it's a normalized value between -1 and 1. The reason why is because it is the result of a dot product. We are actually not dealing with angles at all since we wanted a solely vector approach so that we didn't have to deal with the nature of angles wrapping around and what not.